ArduPlane/commands_logic.cpp

```diff
@@ -117,6 +117,11 @@ bool Plane::start_command(const AP_Mission::Mission_Command& cmd)
     case MAV_CMD_DO_SET_HOME:
         do_set_home(cmd);
         break;
+    case MAV_CMD_DO_SET_MODE:
```

Comment:
this should be covered by a VTOL_LAND to 0,0 lat/lon

Reply:
Hi Tridge, thanks for looking over this
I was unaware vtol_land at 0,0commanded a landing at the current location, does this function the same for vtol take-off? (this is a current pain point for us in that every time we put the plane on the ground, we need to reposition the vtol_takeoff waypoint)